directory/assets/service_checks.json

```diff
@@ -1 +1,11 @@
-[]
+[
+  {
+    "agent_version": "6.17.0",
```

Comment:
```suggestion
    "agent_version": "7.24.0",
```

Reply:
Right, this will be available in next release. Thank you!